yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -43,12 +45,21 @@ import org.apache.spark.util.Utils
 
 class ClientSuite extends SparkFunSuite with Matchers with BeforeAndAfterAll {
 
+  var oldProperties: Properties = null
```

Comment:
Sorry, I made a mistake when I check all of the changes in this file. I will use `ResetSystemProperties` here.